modules/apps/29-fee/types/msgs.go

```diff
@@ -146,12 +162,17 @@ func (msg MsgPayPacketFee) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{signer}
 }
 
-// Route implements sdk.Msg
+// Type implements legacytx.LegacyMsg
+func (msg MsgPayPacketFee) Type() string {
+	return TypeMsgPayPacketFee
+}
+
+// Route implements legacytx.LegacyMsg
 func (msg MsgPayPacketFee) Route() string {
 	return RouterKey
 }
 
-// GetSignBytes implements sdk.Msg.
+// GetSignBytes implements legacytx.LegacyMsg
```

Comment:
I changed it because it looks like `GetSignBytes` is not part of [the `sdk.Msg` interface](https://github.com/cosmos/cosmos-sdk/blob/release/v0.47.x/types/tx_msg.go#L15-L26), but of the [`LegacyMsg` interface](https://github.com/cosmos/cosmos-sdk/blob/release/v0.47.x/x/auth/migrations/legacytx/stdsign.go#L27) instead...

Reply:
either works for me, it will be removed after post eden I guess